types/validator.yaml

```
@@ -111,3 +111,20 @@ SignedAggregateAndProof:
       $ref: "#/AggregateAndProof"
     signature:
       $ref: "./primitive.yaml#/Signature"
+
+Focil:
+ InclusionListDuty:
+  type: object
+  required: [pubkey, validator_index, slot, inclusion_list_committee_root]
+  properties:
+    pubkey:
+      $ref: "./primitive.yaml#/Pubkey"
+    validator_index:
+      $ref: "./primitive.yaml#/Uint64"
+      description: "Index of validator in validator registry."
+    slot:
+      $ref: "./primitive.yaml#/Uint64"
+      description: "The slot at which the validator must propose an inclusion list." 
+    inclusion_list_committee_root:
+      $ref: "./primitive.yaml#/Root"
+      description: "The root of inclusion list committee that the duty is part of" 
```

might wanna rephrase to

```suggestion
      description: "The root of inclusion list committee that the validator is part of." 
```